<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="LeftWindow" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>1000</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="LeftLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="LLLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="LRLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="RightLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QWidget" name="NavBar" native="true">
        <property name="minimumSize">
         <size>
          <width>950</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>950</width>
          <height>65</height>
         </size>
        </property>
        <widget class="QPushButton" name="ExitButton">
         <property name="geometry">
          <rect>
           <x>875</x>
           <y>0</y>
           <width>65</width>
           <height>65</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
        <widget class="QPushButton" name="MinimiseButton">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>0</y>
           <width>65</width>
           <height>65</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="SpellsButton">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>35</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Spells</string>
         </property>
        </widget>
        <widget class="QPushButton" name="InventoryButton">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>35</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Inventory</string>
         </property>
        </widget>
        <widget class="QPushButton" name="CharacterButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>35</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Character</string>
         </property>
        </widget>
        <widget class="QPushButton" name="CombatButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>35</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Combat</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
